### DISCUSSION TITLE
GitHub Copilot in the CLI now in public beta

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#74697] [Copilot News and Announcements]
<img width="697" alt="Screenshot 2023-11-09 at 1 04 05 AM" src="https://github.com/community/community/assets/95834130/196efa09-e1e2-4ec3-b285-a87ae6733558">

We’re excited to announce the launch of a brand new GitHub CLI extension that’s now available as public beta — GitHub Copilot in the CLI. 🎉 

GitHub Copilot in the CLI brings GitHub Copilot right to your terminal, where you can ask it to do things like explain how a command works or suggest a command for a task you want to perform. Learn more about the extension in [our docs](https://docs.github.com/en/copilot/github-copilot-in-the-cli) and provide us your feedback on [our repo](https://github.com/github/gh-copilot/issues).

Learn your way around the command line with GitHub Copilot by your side! 🤖

**Comment by VeryKross** [comment]
I received notification that I was granted access to this. I just wanted to point out that the notification contains a link for "Get started with Copilot in the CLI" which takes you to the Copilot main page where you'll find no mention at all of the CLI feature. Basically a dead end for someone trying to get started.
![image](https://github.com/community/community/assets/11561147/7e2a875a-4e28-4abd-8541-bf49af35bd8c)
